test/template_connector/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.0'
+
+services:
+  # TODO: add a service for the platform you want to connect with secretless
+
+  secretless:
+    image: secretless-broker # this image is built in bin/build
+    volumes:
+      - ./secretless.yml:/secretless.yml
+
+  test:
+    image: secretless-dev # this image is built in bin/build
+    command: go test -v ./test/template_connector
```

Comment:
This current standard is to have a bash `./test` file.  Using go test would be an optional choice each connector integration folder could make.

Perhaps this would be a better standard, but it may not be flexible enough.  In any case, that's a discussion we should have in the channel.